modules/EmailMan/testOutboundEmail.php

```diff
@@ -59,13 +59,12 @@
     $pass = $_REQUEST['mail_smtppass'];
 } elseif (isset($_REQUEST['mail_type'])) {
     $oe = new OutboundEmail();
-    if (is_admin($current_user) && $_REQUEST['mail_type'] == 'system') {
-        $oe = $oe->getSystemMailerSettings();
-    } else {
-        $oe = $oe->getMailerByName($current_user, $_REQUEST['mail_type']);
-    }
-    if (!empty($oe)) {
+    if (!empty($oe) && !empty($_REQUEST["mail_smtpuser"])) {
+        $oe = $oe->getMailerByEmail($_REQUEST["mail_smtpuser"]);
```

Comment:
@jack7anderson7 @JackBuchanan `getMailerByName` may return null. How do we handle such cases?